src/profile/mfa/mfa_factor_sources/trusted_contact_factor_source/trusted_contact_factor_source_contact.rs

```diff
@@ -0,0 +1,62 @@
+use crate::prelude::*;
+
+/// Hints about the trusted contact.
+#[derive(
+    Serialize,
+    Deserialize,
+    Debug,
+    Clone,
+    PartialEq,
+    Eq,
+    Hash,
+    derive_more::Display,
+    uniffi::Record,
+)]
+#[serde(rename_all = "camelCase")]
+#[display("{name} {email_address}")]
+pub struct TrustedContactFactorSourceContact {
```

Comment:
hmm this is not used so much, and that would give it "higher precedence" in autocomplete. So I prefer it if this is longer, more "obscure" if you will, so that we see `TrustedContactFactorSource` when we type "trusted" in IDEs.

Reply:
fair enough